pom.xml

```diff
@@ -168,7 +168,7 @@
       <testjvm.dynamicAgentLoading/>
       <testjvm.jdkSpecificArgs>${testjvm.jigsawArgs} -Dnet.bytebuddy.experimental=true ${testjvm.blockHoundRedefinitionArgs} ${testjvm.dynamicAgentLoading}</testjvm.jdkSpecificArgs>
       <testjvm.extraArgs/>
-      <forkJvmArgs>-Xmx2G ${testjvm.commonArgs} ${testjvm.jdkSpecificArgs} ${testjvm.extraArgs}</forkJvmArgs>
+      <forkJvmArgs>-Djdk.virtualThreadScheduler.parallelism=16 -Xmx4G ${testjvm.commonArgs} ${testjvm.jdkSpecificArgs} ${testjvm.extraArgs}</forkJvmArgs>
```

Comment:
Is increasing to 4GB required to work with virtual threads? Also should we not just use default parallelism? My machine for example will be 32. Or is the idea that CI will be too low at 4?

Reply:
The memory should not be a problem anymore, I'm going to revert it.

We run 8 tests concurrently (`<infinispan.test.parallel.threads>8</infinispan.test.parallel.threads>`) and we have some pinned threads during the JGroups joining. I've configured `virtualThreadScheduler.parallelism` to double that amount to make sure we don't have any issues or random crashes during the test suite.